avc/launch/avc.launch

```diff
@@ -1,9 +1,7 @@
 <launch>
-    <node pkg="avc" type="color_detector_avc" name="color_detector" output="screen"/>
-    <!-- <param name="/use_sim_time" value="true"/> -->
-
     <include file="$(find rr_platform)/launch/transform.launch"/>
     <include file="$(find avc)/launch/mapper.launch"/>
     <include file="$(find avc)/launch/planner.launch"/>
     <include file="$(find rr_description)/speedi/speedi_state_publisher.launch"/>
+    <node pkg="rr_platform" type="scanToPointCloud" name="scanToPointCloud" output="screen" />
```

Comment:
Thanks for moving scanToPointcloud to rr_platform, but this line should still live back in lidar.launch. This node can be run manually when running on bag data. If we start accumulating a lot of nodes that need to be run manually for bag files, we'll make a new launch file for that.